src/main/java/com/twilio/jwt/accesstoken/VoiceGrant.java

```diff
@@ -82,9 +82,9 @@ public Object getPayload() {
          * @param  grant VoiceGrant
          */
         public Payload(VoiceGrant grant) {
-            if (grant.incomingAllow == true) {
+            if (grant.incomingAllow != null) {
               this.incoming = new HashMap<>();
-              this.incoming.put("allow", true);
+              this.incoming.put("allow", grant.incomingAllow);
```

Comment:
Should we do a boolean check or cast here?

Reply:
`VoiceGrant.incomingAllow` is defined as a boolean, so I believe this should only be able to be true, false or null. Is that right?

Reply:
Oh, yup, you're right. I didn't dive deep enough into the chain. 👍